src/Livewire/Product/Product.php

```diff
@@ -58,13 +71,21 @@ public function mount(int $id): void
         $product->append('avatar_url');
 
         $this->product->fill($product);
+        $this->product->product_properties = Arr::keyBy($this->product->product_properties, 'id');
 
         $this->additionalColumns = $product->getAdditionalColumns()->toArray();
+        $this->recalculateDisplayedProductProperties();
+
+        $this->view = data_get(
```

Comment:
This should happen in the render method.
the view is a protected property that means that in case of a next render the property will be default again because mount is only called on the first load of the component.